src/styles/cards.scss

```
@@ -0,0 +1,84 @@
+.cards {
```

Why cards? Block is a singular
```suggestion
.card {
```